distribution/docs/src/main/resources/_contents/_web-applications/admin-console-contents.adoc

```diff
@@ -0,0 +1,201 @@
+====
+
+* Installation Profile Page
+** When a profile is selected, it will auto select applications on the Select Application Page and install them automatically.
+
+** If choose to customize a profile, you will be given the options to manually selected the applications on the Select Application Page.
+
+* In the Select applications to install page, hover over each application to view additional details about the application.
+
+* New applications can be added and existing applications can be upgraded using the Applications Module.
+
+* When an application is selected, dependent applications will automatically be selected.
+
+* When an application is unselected, dependent applications will automatically be unselected.
+
+===== Custom Installation
+
+image::application_details.png[Application Details]
+
+* If apps are preselected when the Select applications to install page is reached, they will be uninstalled if unselected.
+
+* Applications can also be installed using kar deployment as stated in Application Installation.
+
+[WARNING]
+====
+Platform App, Admin App, and Security Services App CANNOT be selected or unselected as it is installed by default and can cause errors if removed.
+
+Security Services App appears to be unselected upon first view of the tree structure, but it is in fact automatically installed with a later part of the installation process.
+====
+
+.General Configuration Page
+image::general_configuration.png[General Configuration,${image.width}]
+
+.General Configuration Page (Certificates)
+image::general_configuration_certs.png[General Configuration Certificates,${image.width}]
+
+[NOTE]
+====
+Certificate information needs to be provided if the host is changed.
+If the `?dev=true` query string was provided, the certificate information will be auto generated using a demo CA
+====
+
+.Final Page
+image::final_page.png[Final Page,${image.width}]
+
+.Shutdown Page
+image::shutdown_page.png[Shutdown Page,${image.width}]
+
+[NOTE]
+====
+The redirect will only work if the certificates are configured in the browser. +
+Otherwise the redirect link must be used.
+====
+
+====== Configuration Module
+
+The configuration module allows administrators to change bundle and service configurations.
+
+====== Set Up the Configuration Module
+
+. Install the module if it is not pre-installed.
+`feature:install admin-modules-configuration`
+
+. Open a web browser and navigate to the ${admin-console} page.
+
+`http://DDF_HOST:DDF_PORT/admin`
+
+. Select the Configurations tab if not already selected.
+
+====== Configurations Tab
+
+==== ${admin-console} Access Control
+
+If you have integrated ${branding} with your existing security infrastructure, then you may want to limit access to parts of the ${branding} based on user roles/groups.
+
+===== Restricting ${branding} Access
+
+. See the documentation for your specific security infrastructure to configure users, roles, and groups.
+
+. Select the Web Context Policy Manager.
+
```

Comment:
Not sure how specific we can be, as it depends on whatever user management they are using. I think I'll just link to the Web Context Policy Manager section in Admin app reference. (That link will be broken until this branch catches up to the changes in the other PRs)